test/config/config.js

```diff
@@ -2,12 +2,18 @@ export default {
   localHTTPServer: {
     protocol: 'http',
     address: 'localhost',
-    port: 9192
+    port: 9192,
+    headers: {
+      'Authorization': 'testjwt'
+    }
   },
   fakeServer: {
     protocol: 'http',
     address: 'thisServerDoesNotExist',
-    port: 9192
+    port: 9192,
+    headers: {
+      'badHeader': 'badHeader'
+    }
```

Comment:
Any reason for the bad header string?  Looks like the test just checks that it matches since the sdk doesn't do anything with headers but pass them through.